clients/admin-ui/src/types/api/models/UploadCustomAsset.ts

```diff
@@ -0,0 +1,6 @@
+import { CustomAssetType } from "./CustomAssetType";
+
+export type UploadCustomAsset = {
+  assetType: CustomAssetType;
+  file: File;
```

Comment:
I definitely appreciate the typing! however, we need this to be autogenerated, otherwise the next time we run `npm run openapi:generate` these types will be overwritten 😢 

`npm run openapi:generate` can only do as well as the python typing in the fastapi docs. I took a peek at https://github.com/ethyca/fidesplus/compare/custom-asset-endpoints and I think that means in that PR, the `key` param has to be a bit more strict—it'll need to be an Enum in order to get something like the `CustomAssetType` you already have. 

if you want types that are only for the FE, then they should live outside of `types/api/models`. Sometimes we have random `type` files in different folders that handle that kind of case.

more info here: https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/types/api/README.md